### ISSUE TITLE
References Section

### ISSUE THREAD

**Issue by andresrcc** [cps-org/cps-wiki#2]
We need a references section. Can build it out of:
[Official CPS wiki References Section](https://github.com/cps-org/cps-wiki/wiki/Welcome)
Asking at project meetings about any extra references.
Once we have a draft of the referenes section, we should be able to close out this issue.

**Comment by andresrcc** [comment]
Agenda item has been added to ask.

**Comment by andresrcc** [comment]
Below is the references section from the official cps repo:

## Reference Documentation

*Facts, specifics, and other information*

Reference documentation for CPS is maintained in [the CPS specification](https://cps-org.github.io/cps/). Since precision and review are important to maintaining a specification, those documents are maintained using a more structured pull request workflow. Please use PRs and issues on [the CPS repo itself](https://github.com/cps-org/cps) to contribute to the CPS specification.

**Comment by andresrcc** [comment]
So far we only have the cps specification. But I'd suggest in this section we should have basic documentation on any cps related tooling as well.

**Comment by bretbrownjr** [comment]
I agree that we should have docs on any tools that support CPS... when that support exists. Short of that, maybe there's a "Getting Involved" process that could be developed that could direct readers to developing projects and tracking tickets for vendors?
